src/data_provider/src/packages/packageLinuxParserHelper.h

```diff
@@ -45,11 +46,11 @@ namespace PackageLinuxHelper
         {
             ret["name"] = info.at("Package");
 
-            std::string priority;
-            std::string groups;
-            std::string multiarch;
+            std::string priority {UNKNOWN_VALUE};
```

Comment:
Why would we have a default value just for some fields?
In `brewWrapper.h` all the fields have one.

Reply:
I narrowed the changes to the ones that fail, you're right.